assets/js/main.js

```diff
@@ -24,15 +32,58 @@ function initDatePicker() {
   }
 }
 
+function initSYAAnalyticsTrack() {
+  Analytics.load();
+
+  // Use document.domain in dev, preview and staging so that tracking works
+  // Otherwise explicitly set the domain as www.gov.uk (and not gov.uk).
+  const cookieDomain = (document.domain === 'www.gov.uk') ? '.www.gov.uk' : document.domain;
+
+  window.GOVUK.analytics = new Analytics({
+    universalId: 'UA-91309785-4',
+    cookieDomain
+  });
+
+  // Set custom dimensions before tracking pageviews
+  // analytics.setDimension(…)
+
+  // Activate any event plugins eg. print intent, error tracking
+  // analyticsPlugins.error();
+  // analyticsPlugins.printIntent();
+
+  // Track initial pageview
+  window.GOVUK.analytics.trackPageview();
```

Comment:
We are already tracking page views via GTM, there is no need to track each page twice!